packages/components/src/components/as-stacked-bar-widget/README.md

```diff
@@ -12,88 +12,79 @@ noSource: true
 lang: html
 showSource: false
 ---
-<as-stacked-bar-widget 
-  show-legend="true"
-  heading="Heading"
-  description="Description">
-</as-stacked-bar-widget>
+<as-stacked-bar-widget id="widget-0" show-legend="true" heading="Star Wars Revenue" description="Description"></as-stacked-bar-widget>
 
 <script>
-    const widget0 = document.querySelector('#widget-0');
-    widget0.data = [{
-        category: 'Category A',
-        values: {
-          v1: 100,
-          v2: 100,
-        }
-      },
-      {
-        category: 'Category B',
-        values: {
-          v1: 100,
-          v2: 200,
-          v3: 300,
-        },
+  const widget0 = document.querySelector('as-stacked-bar-widget');
+  widget0.data = [{
+      category: 'Star Wars Ep. IV: A New Hope',
+      values: {
+        dvd: 27229125,
+        blue: 3555058,
+      }
+    },
+    {
+      category: 'Star Wars Ep. V: The Empire Strikes Back',
+      values: {
+        dvd: 24928640,
+        blue: 2738207,
       },
-      {
-        category: 'Category B',
-        values: {
-          v1: 100,
-          v2: 200,
-          v3: 300,
-          v3: 400,
-        }
+    },
+    {
+      category: 'Star Wars Ep. VI: Return of the Jedi',
+      values: {
+        dvd: 23786454,
+        blue: 1908593,
       }
-    ];
+    }
+  ];
 
-    widget0.metadata = {
-      v1: {
-        label: 'v1-label',
-        color: '#FABADA'
-      },
-      v2: {
-        label: 'v2-label',
-        color: '#C00FEE'
-      },
-      v3: {
-        color: '#ff0088',
-      },
+  widget0.metadata = {
+    dvd: {
+      label: 'Domestic DVD Sales',
+      color: '#33ACEE'
+    },
+    blue: {
+      label: 'Domestic Blu-ray Sales',
+      color: '#EEC649'
     }
+  }
 
-  </script>
+</script>
 ```
 
 
 ### Props
 
 #### **data**: Array<RawStackedBarData>
-The data that will be shown in the stacked bar chart.
+Array of elements containing the data that will be shown in the stacked bar chart.
+
+Each element corresponds to a column and consists of a `category` string that labels the column and an object containing the values of each rectangle in the column.
+
+Each key in the `values` object will be displayed using the same column in every column. Positive values will be stacked in alphabetical order on the top of the zero-axis while negative values will be stacked below the zero axis in reverse alphabetical order.
```

Comment:
Did you mean `the same color in every column`?

Reply:
🙄